reactor-core/src/main/java/reactor/core/scheduler/Schedulers.java

```diff
@@ -839,10 +853,17 @@ public static void shutdownNow() {
 	}
 
 	/**
-	 * {@link Scheduler} that hosts a single-threaded ExecutorService-based worker and is
-	 * suited for parallel work. Will cache the returned schedulers for subsequent calls until dispose.
+	 * The common <em>single</em> instance, a {@link Scheduler} that hosts a single-threaded ExecutorService-based
+	 * worker and is suited for parallel work.
```

Comment:
ooh actually I think the "is suitable for parallel work" is a copy-paste error from `parallel()` from long ago. it sounds misleading to me. care to remove that part while we're at it?

Reply:
Sure, makes sense.